scripts/lint.py

```diff
@@ -203,7 +203,9 @@ def check_rule(self, ctx, rule):
                 continue
 
             try:
-                extractor = capa.main.get_extractor(path, "auto", capa.main.BACKEND_VIV, disable_progress=True)
+                extractor = capa.main.get_extractor(
+                    path, "auto", capa.main.BACKEND_VIV, sigpaths=[], disable_progress=True
```

Comment:
could this lead to differences and hence confusion when running standard (with sigs) vs. lint (without sigs?)

Reply:
good point, i'll register the default sig files here

Reply:
249332a